developer-workflow/sbom.rst

```diff
@@ -107,3 +107,19 @@ When removing a dependency:
    that correct package is removed from the SBOM.
 5. Commit the changes to :cpy-file:`Misc/sbom.spdx.json` and
    :cpy-file:`Tools/build/generate_sbom.py`.
+
+Updating external dependencies (cpython-source-deps)
+----------------------------------------------------
+
+Dependencies for Windows CPython builds are `stored in a separate repository <https://github.com/python/cpython-source-deps>`_
+and then fetched during builds of CPython for Windows in the script :cpy-file:`PCbuild/get_externals.bat`.
+
+In this script the libraries to fetch are designated by ``{name}-{version}`` git refs being added to the ``libraries`` variable.
+SBOM tooling in the CPython repository matches these git refs in order to build the :cpy-file:`Misc/externals.spdx.json`
+SBOM file. When updating external dependencies for a CPython branch:
+
+1. Push the update to the ``cpython-source-deps`` repository and create a new git tag.
+2. Update the entry for the project in ``get_externals.bat``.
+3. Run ``make regen-sbom`` in the CPython source repository.
```

Comment:
Can the people who will likely update Windows dependencies run `make` commands, or should we also give Windows-specific instructions?

Reply:
I would prefer Windows specific instructions.

I believe `PCbuild\build.bat --regen` ought to do it, if that was set up correctly, but I haven't tried it.

Reply:
That hasn't been setup, I didn't realize there were multiple regen flows. I [created an issue](https://github.com/python/cpython/issues/116122) and assigned myself, will submit a PR.

Reply:
I added the command in https://github.com/python/devguide/pull/1280/commits/cefc01d48aaa142585ddde2cef77e66755a02da3